user_guides/jaxws/src/main/jbake/content/config.inc

```diff
@@ -181,55 +250,33 @@ environment variables:
   As an example, the `ts.jte` file contains the property
-  `jaxws.classes.ri`, which contains the classes/jar files that the Jakarta EE 9 Compatible Platform uses. The `jaxws.classes.ri` settings for the Jakarta EE {JakartaEEVersion} Compatible Platform web
-  container are as follows:
+  `jaxws.classes.ri`, which contains the classes/jar files that the Jakarta EE 9
+  Compatible Platform uses. The `jaxws.classes.ri` settings for the Jakarta EE
+  {JakartaEEVersion} Compatible Platform web container are as follows:
 +
 --
 [source,oac_no_warn]
 ----
 jaxws.home.ri=${webcontainer.home.ri}
 jaxws.lib.ri=${jaxws.home.ri}/modules
-endorsed.dirs.ri=${jaxws.home.ri}/modules/endorsed
-
-jaxws.classes.ri=${endorsed.dirs.ri}/webservices-api-osgi.jar:
-${endorsed.dirs.ri}/jaxb-api-osgi.jar:
-${jaxws.lib.ri}/webservices-osgi.jar:
-${jaxws.lib.ri}/jaxb-osgi.jar:
-${jaxws.lib.ri}/gmbal.jar:
-${jaxws.lib.ri}/management-api.jar:
-${jaxws.lib.ri}/mimepull.jar
+jaxws.classes.ri=${jaxws.lib.ri}/webservices-api-osgi.jar${pathsep}
+${jaxws.lib.ri}/jakarta.xml.bind-api.jar${pathsep}
+${jaxws.lib.ri}/jaxb-api-osgi.jar${pathsep}
+${jaxws.lib.ri}/webservices-osgi.jar${pathsep}
+${jaxws.lib.ri}/jaxb-osgi.jar${pathsep}
+${jaxws.lib.ri}/gmbal.jar${pathsep}
+${jaxws.lib.ri}/management-api.jar${pathsep}
+${jaxws.lib.ri}/mimepull.jar${pathsep}
+${jaxws.lib.ri}/ha-api.jar${pathsep}
+${jaxws.lib.ri}/webservices-api.jar${pathsep}
+${jaxws.lib.ri}/jakarta.activation.jar
 ----
 --
 +
 The `jaxws.classes.ri` settings if using the Apache Tomcat web container
-with the Standalone XML-WS 3.0 Compatible Implementation would be as follows:
-+
---
-[source,oac_no_warn]
-----
-jaxws.home.ri=${webcontainer.home.ri}
-jaxws.lib.ri=${jaxws.home.ri}/shared/lib
-endorsed.dirs.ri=${jaxws.home.ri}/common/endorsed
-jaxws.classes.ri=${endorsed.dirs.ri}/jaxws-api.jar:
-${endorsed.dirs.ri}/jaxb-api.jar:
-${jaxws.lib.ri}/FastInfoset.jar:${jaxws.lib.ri}/gmbal-api-only.jar:\
-${jaxws.lib.ri}/ha-api.jar:${jaxws.lib.ri}/jakarta.annotation-api.jar:\
-${jaxws.lib.ri}/jakarta.xml.soap-api.jar:${jaxws.lib.ri}/jaxb-core.jar:\
-${jaxws.lib.ri}/jaxb-impl.jar:${jaxws.lib.ri}/jaxb-jxc.jar:\
-${jaxws.lib.ri}/jaxb-xjc.jar:${jaxws.lib.ri}/jaxws-rt.jar:\
-${jaxws.lib.ri}/jaxwstck.jar:${jaxws.lib.ri}/jaxws-tools.jar:\
-${jaxws.lib.ri}/jsr181-api.jar:${jaxws.lib.ri}/mail.jar:\
-${jaxws.lib.ri}/management-api.jar:${jaxws.lib.ri}/mimepull.jar:\
-${jaxws.lib.ri}/policy.jar:${jaxws.lib.ri}/resolver.jar:\
-${jaxws.lib.ri}/saaj-impl.jar:${jaxws.lib.ri}/sdo-eclipselink-plugin.jar:\
-${jaxws.lib.ri}/stax2-api.jar:${jaxws.lib.ri}/stax-ex.jar:\
-${jaxws.lib.ri}/streambuffer.jar:${jaxws.lib.ri}/woodstox-core-asl.jar:
-----
---
-+
-  e.  Verify that the `tools.jar` property is set to the location of the
-  `tools.jar` file that is distributed with Java SE.
-  f.  Set the `impl.vi`, `impl.vi.deploy.dir`, `impl.vi.host`, and
+with the Standalone XML-WS 3.0 Compatible Implementation should contain
```

Comment:
consider replacing all references of JAX-WS with {TechnologyShortName}

Reply:
TechnologyShortName is "XML Web Services".  But yes, I'll try it with combination of TechnologyVersion too.

Reply:
Done